modules/services/window-managers/bspwm.nix

```diff
@@ -0,0 +1,138 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+	cfg = config.xsession.windowManager.bspwm;
```

Comment:
Please make sure to use two spaces per indentation level, which is sort-of standard within the Nix community.

Reply:
Okay, I'll do that in the following commit